frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -909,7 +909,7 @@ export const AlertConfigurationCard = ({
 							<p>
 								Add webhook destinations for this alert, sent as
 								JSON over HTTP. See the{' '}
-								<Link to="https://www.highlight.io/docs/general/product-features/general-features/alerts/webhooks">
+								<Link to="https://www.highlight.io/docs/general/product-features/general-features/webhooks">
```

Comment:
Drive by bug fix to update to this URL
https://www.highlight.io/docs/general/product-features/general-features/webhooks

The previous URL is broken.